masq/tests/startup_shutdown_tests_integration.rs

```diff
@@ -60,6 +60,30 @@ fn masq_propagates_errors_related_to_default_terminal_integration() {
     );
 }
 
+#[test]
+fn masq_terminates_in_reaction_to_loss_of_the_connection_with_daemon_or_node_integration() {
```

Comment:
This test name is a little misleading. What this test proves is that when `masq` is connected to the Daemon, and the Daemon goes down, `masq` terminates. The Node isn't involved here.